ui/js/compareperf.js

```diff
@@ -10,7 +10,7 @@ perf.controller('CompareChooserCtrl', [
                               ThRepositoryModel, ThResultSetModel) {
     ThRepositoryModel.get_list().success(function(projects) {
       $scope.projects = projects;
-      $scope.originalProject = $scope.newProject = projects[0];
+      $scope.originalProject = $scope.newProject = projects[1];
```

Comment:
Ok, after some investigation I found out that we should probably be using thDefaultRepo here. :) Just set the value to that here, we'll assume that it's always in the repository list (a fair assumption IMO, we do this elsewhere in treeherder).

https://github.com/mozilla/treeherder/blob/master/ui/js/values.js#L127

You can see an example of using it in this file:

https://github.com/mozilla/treeherder/blob/master/ui/js/controllers/main.js#L12
https://github.com/mozilla/treeherder/blob/master/ui/js/controllers/main.js#L331